configurationfiles_test.go

```diff
@@ -0,0 +1,38 @@
+package codacytool
+
+import (
+	"testing"
+)
+
+func TestGetBasePath(t *testing.T) {
+	basePath := getBasePath()
+	if basePath != testsResourcesLocation {
+		t.Errorf("Expected: %s; Got: %s", _toolFilesBasePath, basePath)
```

Comment:
can we make use of an `assert`ion library, or something alike, and avoid to prettify the error messages by hand?

Reply:
Nice suggestion 👌
Added testify library to make assertion like tests instead of prettify the error messages by hand.